src/test/java/com/pages/Homepage.java

```diff
@@ -0,0 +1,48 @@
+package com.pages;
```

Comment:
src/test should only contain test files both pages and steps should be in src/main